bundles/org.openhab.binding.orbitbhyve/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="orbitbhyve"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Mode Channel Type -->
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Irrigation Mode</label>
+		<description>Channel representing mode of Orbit B-Hyve Device (auto/manual)</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="next_start">
+		<item-type>DateTime</item-type>
+		<label>Next Watering Start Time</label>
+		<description>Channel representing start time of the next watering</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="rain_delay">
+		<item-type>Integer</item-type>
+		<label>Rain delay</label>
+		<description>Channel representing rain delay in hours</description>
+		<state pattern="%d h"></state>
+	</channel-type>
+	<channel-type id="watering_time">
+		<item-type>Integer</item-type>
+		<label>Zone watering time</label>
+		<description>Channel representing the manual zone watering time in minutes</description>
+		<state max="240" min="0" pattern="%d min"></state>
+	</channel-type>
+	<channel-type id="control">
+		<item-type>Switch</item-type>
+		<label>Sprinkler State Control</label>
+		<description>Channel for enabling/disabling the sprinkler (ON/OFF)</description>
+	</channel-type>
+	<channel-type id="smart_watering">
+		<item-type>Switch</item-type>
+		<label>Sprinkler Smart Watering Control</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Made it shorter